lib/web/userpreferences.go

```diff
@@ -0,0 +1,87 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/gravitational/trace"
+	"github.com/julienschmidt/httprouter"
+
+	"github.com/gravitational/teleport/api/gen/proto/go/userpreferences/v1"
+	"github.com/gravitational/teleport/lib/httplib"
+)
+
+type AssistUserPreferencesResponse struct {
+	PreferredLogins []string                         `json:"preferred_logins"`
```

Comment:
Looks like assist (and a few other cases) uses snake_case in our web API's JSON tags, but most things use camelCase. Not a huge issue, but the inconsistency makes us more prone to error.

Reply:
Ah I thought we preferred snake_case. I'll use camelCase here, it simplifies the frontend types.